percy/__init__.py

```diff
@@ -19,3 +19,10 @@ def percySnapshot(browser, *a, **kw):
     def percy_screenshot(driver, *a, **kw):
         raise ModuleNotFoundError("[percy] `percy-appium-app` package is not installed, "\
                         "please install it to use percy_screenshot command")
+
+try:
+    from percy.snapshot import percy_automate_screenshot
```

Comment:
lets discuss the naming once with Product, we were suppose to keep `percy_screenshot` only

Reply:
Then how are we supposed to handle appium as currently we import percy_screenshot from appium package even someone imported percy selenium. then this behaviour will be altered in future release